src/main/java/core/basesyntax/service/FruitsFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FruitsFromFile {
+    List<String> getFruits(List<FruitTransaction> fruitTransactions);
+}
```

Comment:
The interface name 'FruitsFromFile' is a bit misleading since it suggests that it deals with obtaining fruits from a file, but the method inside suggests that it's converting `FruitTransaction` objects into a list of strings. Consider renaming the interface to better reflect its responsibility, such as 'FruitTransactionConverter' or similar.